EPRN.UnitTests/Portal/Services/WasteServiceTests.cs

```diff
@@ -387,5 +390,39 @@ public async Task SaveTonnage_NullTonnage_ThrowsNullReferenceException()
                 It.IsAny<int>(),
                 It.IsAny<double>()), Times.Never);
         }
+
+        [TestMethod]
+        public async Task SaveBaledWithWireModel_Succeeds_WithValidModel()
+        {
+            // Arrange
+            BaledWithWireModel baledWithWireModel = new BaledWithWireModel();
+            baledWithWireModel.JourneyId = 1;
+            baledWithWireModel.BaledWithWire = true;
+
+
+            // Act
+            await _wasteService.SaveBaledWithWire(baledWithWireModel);
+
+            // Assert
+            _mockHttpWasteService.Verify(s => s.SaveBaledWithWire(
+                It.Is<int>(p => p == 1),
+                It.Is<bool>(p => p == true)
+            ));
```

Comment:
Also you could tag on how many times this mocked method should be called. Will be a good assertion if someone removes the call or adds an additional call.